drv/sprot-api/src/lib.rs

```diff
@@ -495,4 +527,120 @@ pub struct SprotIoStats {
     pub sp: SpIoStats,
 }
 
+impl SpRot {
```

Comment:
I didn't know you could do this without having the type imported/visible. I find it somewhat disconcerting, but I guess its fine ™️ 

Thank you for adding the method though!